Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -4,25 +4,24 @@ import Keys
 
 protocol WeatherForCityViewModel {
```

Comment:
Could you add description to protocol, properties and functions?

Reply:
👍